src/sidecar/checkArchitecture.ts

```diff
@@ -42,6 +43,8 @@ export class PlatformArch {
     let os = this.platform;
     if (os === "darwin") {
       os = "OS X (Darwin)";
+    } else if (os === "win32") {
+      os = "Windows";
```

Comment:
New tests over toString() demonstrated we'd say "Win32 on x64" which read poorly.